src/pdf/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -147,23 +147,43 @@ const LabelCard: React.FC<LabelCardProps> = ({ data, index, quantity }) => {
 };
 
 export interface ShippingLabelsPdfProps {
-    data: ShippingLabelData;
+    data: ShippingLabelData[];
+}
+
+interface ShippingLabelsForSingleParcelProps {
+    parcelDataForShippingLabel: ShippingLabelData;
 }
 
+const ShippingLabelsForSingleParcel: React.FC<ShippingLabelsForSingleParcelProps> = ({
+    parcelDataForShippingLabel,
+}) => {
+    return (
+        parcelDataForShippingLabel.label_quantity > 0 &&
+        [...Array(parcelDataForShippingLabel.label_quantity)].map((_, index: number) => {
```

Comment:
i think this is equivalent?
```suggestion
        Array(parcelDataForShippingLabel.label_quantity).map((_, index: number) => {
```

Reply:
Doesn't seem to generate the pdf with the change so I'm going to leave it as it is 😕